docs/snippets/angular/button-story-component-args-primary.ts.mdx

```diff
@@ -16,5 +13,5 @@ export default {
     //👇 Now all Button stories will be primary.
     primary: true,
   },
-} as Meta;
```

Comment:
are we getting rid of these types in the TS snippets?

Reply:
or temporary until we figure out the TS story?

Reply:
For the Angular ones, I'm dropping the Meta type. React TS and TS snippets in the common folder I'm leaving them as is until we figure out the TS story. I'll push a commit with this in a bit